components/ctrl/src/ctrl.c

```diff
@@ -11,20 +11,22 @@
 #include <opencan_tx.h>
 #include <selfdrive/pid.h>
 
-#define ENCODER0_CHAN_A 4
-#define ENCODER0_CHAN_B 3
+#define ENCODER0_CHAN_A 3
+#define ENCODER0_CHAN_B 4
 #define ENCODER1_CHAN_A 36
 #define ENCODER1_CHAN_B 35
 
 #define ESP_INTR_FLAG_DEFAULT 0
 
-#define ENCODER_MAX_TICKS	   600	// slightly over 5MPH
-#define ENCODER_TICKS_PER_ROTATION 4000
+#define ENCODER_MAX_TICKS	   777	// slightly over 10MPH
```

Comment:
```suggestion
#define ENCODER_MAX_TICKS	   400	// slightly over 5mph
```
We should put this back to our regular value for safety reasons.